pkg/dashboards/kubernetes/proxy/proxy.go

```diff
@@ -0,0 +1,123 @@
+package proxy
+
+import (
+	"github.com/perses/community-dashboards/pkg/dashboards"
+	panelsGostats "github.com/perses/community-dashboards/pkg/panels/gostats"
+	panels "github.com/perses/community-dashboards/pkg/panels/kubernetes"
+	"github.com/perses/community-dashboards/pkg/promql"
+	"github.com/perses/perses/go-sdk/dashboard"
+	panelgroup "github.com/perses/perses/go-sdk/panel-group"
+	listVar "github.com/perses/perses/go-sdk/variable/list-variable"
+	labelValuesVar "github.com/perses/plugins/prometheus/sdk/go/variable/label-values"
+)
+
+func withProxyStatsGroup(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Proxy Status",
+		panelgroup.PanelsPerLine(1),
+		panelgroup.PanelHeight(8),
+		panels.ProxyUpStatus(datasource, labelMatcher),
+	)
+}
+
+func withProxyRulesSyncRateGroup(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Proxy Rules Sync Rate",
+		panelgroup.PanelsPerLine(2),
+		panelgroup.PanelHeight(8),
+		panels.RulesSyncRate(datasource, labelMatcher),
+		panels.RulesSyncLatency(datasource, labelMatcher),
+	)
+}
+
+func withProxyNetworkProgrammingRateGroup(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Proxy Network Programming Rate",
+		panelgroup.PanelsPerLine(2),
+		panelgroup.PanelHeight(8),
+		panels.NetworkProgrammingRate(datasource, labelMatcher),
+		panels.NetworkProgrammingLatency(datasource, labelMatcher),
+	)
+}
+
+func withProxyKubeAPIRequestsGroup(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	// TODO(saswatamcode): Add a way to configure these.
+	labelMatchersToUse := []promql.LabelMatcher{
+		promql.ClusterVar,
+		promql.InstanceVar,
+		{
+			Name:  "job",
+			Value: "kube-proxy",
+			Type:  "=",
+		},
+	}
+
+	labelMatchersToUse = append(labelMatchersToUse, labelMatcher)
+
+	return dashboard.AddPanelGroup("Kube API Requests",
+		panelgroup.PanelsPerLine(3),
+		panelgroup.PanelHeight(8),
+		panels.KubeAPIRequestRate(datasource, labelMatchersToUse...),
+		panels.PostRequestLatency(datasource, labelMatchersToUse...),
+		panels.GetRequestLatency(datasource, labelMatchersToUse...),
+	)
+}
+
+func withProxyResources(datasource string, clusterLabelMatcher promql.LabelMatcher) dashboard.Option {
+	// TODO(saswatamcode): Add a way to configure these.
+	labelMatchersToUse := []promql.LabelMatcher{
+		promql.ClusterVar,
+		promql.InstanceVar,
+		{
+			Name:  "job",
+			Value: "kube-proxy",
+			Type:  "=",
+		},
+	}
+
+	labelMatchersToUse = append(labelMatchersToUse, clusterLabelMatcher)
+
+	return dashboard.AddPanelGroup("Resource Usage",
+		panelgroup.PanelsPerLine(2),
+		panelgroup.PanelHeight(8),
```

Comment:
Suggestion(Non Blocking): I also feel like the PanelHeight(8) looks good and maybe we shall change the default PanelHeight to `8`? WDYT?  
```
		panelgroup.PanelHeight(8),
```

Reply:
Yup, it's in Perses sdk right?